S17-procasync/basic.t

```diff
@@ -106,3 +106,12 @@ throws-like { Proc::Async.new }, X::Multi::NoMatch,
     is $captured, "Hello World\n",
         "Tapping stdout supply after start of process does not lose data";
 }
+
+$pc = Proc::Async.new("perl6", "-e ''"); # It taps a warning, but it is safe to catch.
```

Comment:
Sure! I didn't know what command is better to write, since things like "ls" are quite UNIX-like-specific.